include/unifex/single_thread_context.hpp

```diff
@@ -38,6 +38,10 @@ class context {
   auto get_scheduler() noexcept {
     return loop_.get_scheduler();
   }
+
+  std::thread::id get_id() const noexcept {
```

Comment:
Should this be `get_thread_id`?
```suggestion
  std::thread::id get_thread_id() const noexcept {
```